Automattic/sensei
# Add installer and student progress schema

Fixes #5445 

### Changes proposed in this Pull Request

* Add the installer class (without data migrator here).
* Add schema class with student progress table.

### Testing instructions
- Connect to your database.
- Make sure you don't have the `wp_sensei_lms_progress` table.
- Make sure you don't have `sensei_lms_version` option (`SELECT * FROM wp_options WHERE option_name="sensei_lms_version" LIMIT 1;`). Delete it if you have (`DELETE FROM wp_options WHERE option_name="sensei_lms_version";`).
- Open any page of the website.
- Check your database. We expect the `wp_sensei_lms_progress` table exists and is empty.


### New/Updated Hooks

* `sensei_lms_installed` - Fires after the installation completes.
* `sensei_lms_schema_get_tables` - Filter the list of known tables. If plugins need to add new tables, they can inject them here.
